test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,8 @@
 const { joinNames } = require("../src");
 
+const inputNames = [{name: "Bart"}, {name: "Lisa"}, {name: "Maggie"}];
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  test("returns string of names, seperated by commas and an ampersand", () => {
+    expect(joinNames(inputNames)).toEqual("Bart, Lisa & Maggie");
```

Comment:
We definitely need another test case here as this function should be able to handle arrays of any number of names from 1 upwards. This could be re-written like
```
  test("returns string of names, seperated by commas and an ampersand", () => {
    expect(
      joinNames([{ name: "Bart" }, { name: "Lisa" }, { name: "Maggie" }])
    ).toEqual("Bart, Lisa & Maggie");
    expect(joinNames([{ name: "Homer" }, { name: "Barney" }])).toEqual(
      "Homer & Barney"
    );
    expect(joinNames([{ name: "Homer" }])).toEqual("Homer");
  });
```
You could then remove the `inputNames` array at the top too